### ISSUE TITLE
Latest v9.5 release is infected with a Trojan!!!

### ISSUE THREAD

**Issue by innerspacedev** [nanopool/Claymore-Dual-Miner#24]
The Latest v9.5 release is infected with a Trojan!!!

**Comment by ryanrca** [comment]
Can I have some more information?  Can you point in the source code where this trojan lives?  Where did you get your version of claymore?  Was it directly from github?

**Comment by innerspacedev** [comment]
Yes I downloaded it from Github and scanned it with Bitdefender.
I haven’t looked at the source code, but it’s definitely in the .exe
Mark :-)

**Comment by Glockenbeat** [comment]
Can confirm on Win10 with Kaspersky Internet Security alerting on EthDcrMiner64.exe with heuristic positive HEUR:Trojan.Win32.Generic.

**Comment by rlightner** [comment]
Sounds like a false positive.

**Comment by computeronix** [comment]
A chunk of AVs are now flagging various miners just because the miners have been included in known attacks.  By doing this it will alert the system they have this on their machine.  Adding an exclusion/whitelisting should resolve.  (Recent example: https://news.bitcoin.com/hackers-use-nsa-exploit-to-mine-monero-using-victims-computers/)
